src/utils/contracts.ts

```diff
@@ -99,6 +99,19 @@ export const METADATA_RENDERER_ADDRESS: {[key in Environment]: string} = {
   [Environment.mainnet]: '0x60B839C2f7dBa29eB93b094067E6C87067d1B3df'.toLowerCase(),
 }
 
+export const ERC20_HLG_ADDRESSES: {[key in Environment]: string} = {
```

Comment:
Where did you grab these addresses from? They're not matching what I have in the protocol repo

Reply:
I'm not entirely sure, but I believe Angello provided me with those addresses a few months back.